### DISCUSSION TITLE
Error uploading to github

### DISCUSSION THREAD

**Discussion by Baba-Coder** [community/community#128074] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

![pics](https://github.com/community/community/assets/42784925/9708037a-9522-4bf7-af1a-9bd6de7be07a)

**Comment by vishal11u** [comment]
Response #1

**Comment by grandmas-favorite** [comment]
HI @Baba-Coder!

Since this issue isn't directly related to Actions, I'm going to push this Discussion over to the General category where you're more likely to get a useful response 😄

**Reply by shahidhasan66ss** [reply]
what is the solution for this error?
